crates/hc_sandbox/tests/cli.rs

```diff
@@ -225,6 +249,7 @@ fn get_sandbox_command() -> Command {
 async fn get_launch_info(stdout: &mut ChildStdout) -> LaunchInfo {
     let mut lines = BufReader::new(stdout).lines();
     while let Ok(Some(line)) = lines.next_line().await {
+        println!("@@@@@-{line}-@@@@@");
         if let Some(index) = line.find("#!0") {
```

Comment:
I've also been curious about this line, where does this string come from?

Reply:
*This* one is a magic string... It is output by conductor itself, right?